lib/code_fund_web/slack.ex

```diff
@@ -0,0 +1,45 @@
+defmodule CodeFundWeb.SlackBot do
+  require Logger
+  use Slack
+
+  @doc """
+  Sends message to channel use Web api.
+  """
+  @spec web_send_message(String.t()) :: any
+  def web_send_message(message) do
+    token = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_token]
+
+    if token do
+      channel = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_channel]
+      avatar = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_avatar]
+      bot_name = Application.get_env(:code_fund, CodeFundWeb.Endpoint)[:slack_bot_name]
+
+      Slack.Web.Chat.post_message(channel, message, %{
+        icon_url: avatar,
+        username: bot_name
+      })
+    end
+  end
+
+  def handle_connect(slack, state) do
+    Logger.debug(fn -> "Connected as #{slack.me.name}" end)
```

Comment:
We have a logger interface that connects to Rollbar that we'd like to use in cases like this -- Check the `CodeFund.Reporter` module.

Reply:
But this isn't error report, it's just debug info
btw, all the handle_xxx methods here - it is skeleton for interactive bot.
for simple sending of message to channel it is enough to do http get request...

Reply:
That's fine, we can still log debug messages out to Rollbar but might require another clause in the Reporter. Rollbar is currently our long term log storage so anything critical or that deals w/ external services we want to log out i'd like to have there if possible.